bigbone/src/test/assets/featured_tags_post_error_422.json

```diff
@@ -0,0 +1,3 @@
+{
+  "error": "Validation failed: Tag is invalid"
+}
```

Comment:
The exact content of the errors is not yet evaluated in tests. This will be possible once #282 is implemented. Already adding the JSON file here.